README.md

```diff
@@ -195,23 +235,12 @@ To share more information, example:
 
 ```javascript
     var fb = require('facebook');
-    if(fb.getCanPresentShareDialog()) { //checks to see if facebook app installed
```

Comment:
yup. facebook API will determine if the dialog is presented in the Facebook native app or the web browser.